lib/collector/facts.js

```diff
@@ -30,6 +36,17 @@ function facts(agent, callback) {
       const systemInfo = data.systemInfo || Object.create(null)
       const environment = data.environment || []
 
+      const packages = data.environment.find((elt) => elt[0] === 'Packages')[1]
+      const dedupPackages = new Set()
+      packages.forEach((dep) => dedupPackages.add(JSON.parse(dep)[0]))
```

Comment:
we talked about this in slack. I don't think it matters we only increment the call count once per agent start for a given package.  also i recommend extracting this little bit of code to its own function so its clear the intention

Reply:
After thinking about this a bit more, the dedup feature of the `Set` here is kind of secondary compared to the slight gain of doing O(1) lookups to see if things are in the set.

I'll write a `packagesToSet` function that JSON parses each entry of the array to get the package name and puts it in a set.